### ISSUE TITLE
pycapi does not work on Python 3.8

### ISSUE THREAD

**Issue by Marco-Sulla** [brandtbucher/pycapi#2]
```shell
(venv_3_8) marco@buzz:~/sources/venv_3_8$ python
Python 3.8.2 (tags/v3.8.2-dirty:7b3ab5921f, Mar  1 2020, 16:16:55) 
[GCC 9.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
```
```python
>>> import pycapi
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: /home/marco/.local/share/virtualenvs/venv_3_8-5n31ha3c/lib/python3.8/site-packages/pycapi.cpython-38-x86_64-linux-gnu.so: undefined symbol: PyEval_ReInitThreads
```

`PyEval_ReInitThreads` was removed in Python 3.8.

**Comment by brandtbucher** [comment]
Thanks for the heads-up. I'll get this fixed soon, and add a CI job for 3.8.

**Comment by tom66** [comment]
Is this issue resolved?  Still having this problem on Python 3.8.3.  Thanks

**Comment by brandtbucher** [comment]
Sorry, this slipped through the cracks. I'll have a fix out today!

**Comment by brandtbucher** [comment]
This is fixed in 0.81.1. Let me know if you encounter any other issues!
